client/src/luigi-client.js

```diff
@@ -410,6 +419,30 @@ const LuigiClient = {
           { msg: 'luigi.set-page-dirty', dirty: isDirty },
           '*'
         );
+      },
+      /**
+       * Shows a confirmation modal.
+       * @param {Object} content the content of the confirmation modal
+       * @param {string} content.header text for modal header
+       * @param {string} content.body text for modal body
+       * @param {string} content.buttonConfirm text for modal confirm button
+       * @param {string} content.buttonDismiss text for modal dismiss button
```

Comment:
```suggestion
       * @param {string} content.buttonDismiss the label for the modal dismiss button
```

Reply:
I thought that maybe in the case of a button `label` will be better? But it's not a must.